content/php/concepts/string-functions/terms/html-entity-decode/html-entity-decode.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'PHP html_entity_decode() Function'
+Description: 'Converts HTML entities to characters.'
+Subjects: 
+- 'Web Design'
+- 'Web Development'
+Tags:
+- 'Functions'
+- 'Characters'
+CatalogContent:
+- 'php/concepts/string-functions/terms/html-entity-decode'
+
+## About this function
+
+html_entity_decode() is the opposite of htmlentities() in that it converts HTML entities in the string to their corresponding characters.
+
+## SyntaxSection
+```pseudo
+html_entity_decode(str, flags, character-set)
+
```

Comment:
explain what is `str`, `flags` and `charater-set` in the above syntax code.

Reply:
@dakshdeepHERE There is an explanation above the line: "## Example". Is it enough or I did not underestand?